### PR TITLE
Some additions to the OS documentation

### PR DESCRIPTION
**Pull request by hurikhan**
^^

### CODE REVIEW

File: doc/base/classes.xml [hunk 1]
```diff
@@ -22372,12 +22374,17 @@
 			<argument index="0" name="file" type="String">
 			</argument>
 			<description>
+				Dumps the memory allocation ringlist to a file(only works in debug).
```
Comment (Calinou) [inline doc/base/classes.xml]: Add a space before opening parenthesis.

File: doc/base/classes.xml [hunk 2]
```diff
@@ -22372,12 +22374,17 @@
...
+				Entry format per line: "Address - Size - Description"
 			</description>
 		</method>
 		<method name="dump_resources_to_file">
 			<argument index="0" name="file" type="String">
 			</argument>
 			<description>
+				Dumps all used resources to file(only works in debug).
```
Comment (Calinou) [inline doc/base/classes.xml]: Add a space before opening parenthesis.

File: doc/base/classes.xml [hunk 3]
```diff
@@ -22583,6 +22604,7 @@
 			<argument index="0" name="code" type="int">
 			</argument>
 			<description>
+				Returns the given scancode as a string(e.g. Return values: "Escape", "Shift+Escape").
```
Comment (Calinou) [inline doc/base/classes.xml]: Add a space before opening parenthesis.

**Comment by hurikhan** [comment]
@Calinou you are right.

**Pull request merged**